feat(cmd): improve UI on multi-node command

julienrbrt opened the pull request:
Improve UI on the multi-node command.

ref: https://github.com/ignite/cli/issues/4374#issuecomment-2687663650

Before:

![image](https://github.com/user-attachments/assets/03c5bb13-8f55-43c3-b66e-37335ade20ab)

After: 

![image](https://github.com/user-attachments/assets/6d883ce0-6a27-47f5-a35a-1c73b7ea7cda)

https://asciinema.org/a/716723

Pantani commented:
The UI is excellent, and the interaction is easy! One thing to improve, we need to start the latest validator (4) before we start the others. Maybe we can have a command to begin all, and also some instructions about which one is the primary node

julienrbrt commented:
> The UI is excellent, and the interaction is easy! One thing to improve, we need to start the latest validator (4) before we start the others. Maybe we can have a command to begin all, and also some instructions about which one is the primary node

Updated! Now they all starts at once, and i've updated the default template to have only 3 nodes by default.

julienrbrt commented:
@Pantani this is R4R again :)
